Traducción: Elemento HTML - Title (#6253)
mdn/translated-content: files/es/web/html/element/title/index.md

### Títulos de página y SEO

El contenido del título de una página puede tener implicaciones significativas para la optimización de motores de búsqueda ({{glossary("SEO")}}). En general, un título descriptivo más largo funciona mejor que los títulos cortos o genéricos. El contenido del título es uno de los componentes que utilizan los algoritmos de los motores de búsqueda para decidir el orden en el que enumerar las páginas en los resultados de búsqueda. Además, el título es el "gancho" inicial mediante el cual captas la atención de los lectores que miran la página de resultados de búsqueda.

Algunas pautas y consejos para componer buenos títulos:

- Evite los títulos de una o dos palabras. Utilice una frase descriptiva o una combinación de términos y definiciones para las páginas de glosario o estilo de referencia.
- Los motores de búsqueda suelen mostrar los primeros 55 a 60 caracteres del título de una página. El texto más allá de eso puede perderse, así que trate de no tener títulos más largos que eso. Si debe usar un título más largo, asegúrese de que las partes importantes aparezcan antes y que no haya nada crítico en la parte del título que es probable que se elimine.

> - No utilice "Blobs de palabras clave." Si su título es solo una lista de palabras, los algoritmos a menudo reducen la posición de su página en los resultados de búsqueda.

Reviewer: Para indicar que no es una palabra nativa, también podría quedar "listas de palabras clave", ya que la interpretación de a lo que se refiere es algo similar

```suggestion
- No utilice "_blobs_ de palabras clave." Si su título es solo una lista de palabras, los algoritmos a menudo reducen la posición de su página en los resultados de búsqueda.
```

- Trate de asegurarse de que sus títulos sean lo más únicos posible dentro de su propio sitio. Los títulos duplicados, o casi duplicados, pueden contribuir a resultados de búsqueda inexactos.

## Ejemplo

```html

> <title>Awesome interesting stuff</title>

Reviewer: Los textos literales también pueden ser traducidos para ayudar al lector

```suggestion
<title>Cosas increíbles e interesantes</title>
```

```

> Este ejemplo establece una página cuyo título (tal como se muestra en la parte superior de la ventana o en la pestaña de la ventana) es "Awesome interesting stuff".

Reviewer: Para concordar con el ejemplo

```suggestion
Este ejemplo establece una página cuyo título (tal como se muestra en la parte superior de la ventana o en la pestaña de la ventana) es "Cosas increíbles e interesantes".
```

## Problemas de accesibilidad

Es importante proporcionar un título preciso y conciso para describir el propósito de la página.

Una técnica de navegación común para los usuarios de tecnología de asistencia es leer el título de la página e inferir el contenido que contiene. Esto se debe a que navegar en una página para determinar su contenido puede ser un proceso lento y potencialmente confuso. Los títulos deben ser únicos para cada página de un sitio web, idealmente mostrando primero el propósito principal de la página, seguido del nombre del sitio web. Seguir este patrón ayudará a garantizar que el lector de pantalla anuncie primero el propósito principal de la página. Esto brinda una experiencia mucho mejor que tener que escuchar el nombre de un sitio web antes del título único de la página, para cada página a la que navega un usuario en el mismo sitio web.

### Ejemplo

```html

> <title>Menu - Blue House Chinese Food - FoodYum: Online takeout today!</title>

Reviewer: Los textos literales también pueden ser traducidos para ayudar al lector

```suggestion
<title>Menú - Comida china Blue House - FoodYum: ¡Comida a domicilio en línea hoy!</title>
```

```

Si el envío de un formulario contiene errores y el envío vuelve a representar la página actual, el título se puede usar para ayudar a que los usuarios se den cuenta de cualquier error en su envío. Por ejemplo, actualice el valor de `title` de la página para reflejar cambios significativos en el estado de la página (como problemas de validación de formularios).

### Ejemplo

```html

> <title>2 errores - Tu orden - Blue House Chinese Food - FoodYum: Online takeout today!</title>

Reviewer: ```suggestion
<title>2 errores - Tu orden - Comida china Blue House - FoodYum: ¡Comida a domicilio en línea hoy!</title>
```

```

> > **Nota:** Actualmente, los lectores de pantalla no anunciarán automáticamente la actualización dinámica del título de una página. Si va a actualizar el título de la página para reflejar cambios significativos en el estado de una página, entonces también puede ser necesario el uso de [ARIA Live Regions](/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions).

Reviewer: - Agregar los vinculos en `/es/` el bot automáticamente dirigirá a español si esta disponible o inglés en caso contrario
- Se traduce lo de ARIA para ayudar al lector

```suggestion
> **Nota:** Actualmente, los lectores de pantalla no anunciarán automáticamente la actualización dinámica del título de una página. Si va a actualizar el título de la página para reflejar cambios significativos en el estado de una página, entonces también puede ser necesario el uso de [regiones en vivo de ARIA](/es/docs/Web/Accessibility/ARIA/ARIA_Live_Regions).
```

> - [MDN Understanding WCAG, Guideline 2.4 explanations](/en-US/docs/Web/Accessibility/Understanding_WCAG/Operable#guideline_2.4_—_navigable_provide_ways_to_help_users_navigate_find_content_and_determine_where_they_are)

Reviewer: ```suggestion
- [MDN Entendiendo las WCAG, Directriz 2.4 explicaciones](/es/docs/Web/Accessibility/Understanding_WCAG/Operable#guideline_2.4_—_navigable_provide_ways_to_help_users_navigate_find_content_and_determine_where_they_are)
```

> - [Understanding Success Criterion 2.4.2 | W3C Understanding WCAG 2.1](https://www.w3.org/WAI/WCAG21/Understanding/page-titled.html)

Reviewer: ```suggestion
- [Entendiendo el Criterio de Conformidad 2.4.2 | W3C Entendiendo las WCAG 2.1](https://www.w3.org/WAI/WCAG21/Understanding/page-titled.html)
```

## Especificaciones

{{Specifications}}

> ## Compatibilidad del navegador

Reviewer: Para homologar este apartado en las traducciones

```suggestion
## Compatibilidad con navegadores
```
